src/game/stage/collect_ring_effect.c

```diff
@@ -0,0 +1,82 @@
+#include "global.h"
+#include "task.h"
+#include "lib/m4a.h"
+#include "game/game.h"
+#include "game/stage/rings_manager.h"
+
+#include "constants/animations.h"
+
+typedef struct {
+    Sprite s;
+    u8 unk30;
+    u8 unk31;
+} Struct_RingEffect;
```

Comment:
Any reason to name `Struct_`?

Reply:
No special reason, I've just started doing it at some point. Could change that if you wanna.